doc/releases/squid.rst

```diff
@@ -0,0 +1,561 @@
+=====
+Squid
+=====
+
+Squid is the 19th stable release of Ceph.
+
+v19.2.0 Squid
+=============
+
+Ceph
+~~~~
+
+* ceph: a new `--daemon-output-file` switch is available for `ceph tell`
+  commands to dump output to a file local to the daemon. For commands which
+  produce large amounts of output, this avoids a potential spike in memory
+  usage on the daemon, allows for faster streaming writes to a file local to
+  the daemon, and reduces time holding any locks required to execute the
+  command. For analysis, it is necessary to manually retrieve the file from the host
+  running the daemon. Currently, only ``--format=json|json-pretty``
+  are supported.
+* ``cls_cxx_gather`` is marked as deprecated.
+* Tracing: The blkin tracing feature (see
+  https://docs.ceph.com/en/reef/dev/blkin/) is now deprecated in favor of
+  Opentracing
+  (https://docs.ceph.com/en/reef/dev/developer_guide/jaegertracing/) and will
+  be removed in a later release.
+* PG dump: The default output of ``ceph pg dump --format json`` has changed.
+  The default JSON format produces a rather massive output in large clusters
+  and isn't scalable, so we have removed the 'network_ping_times' section from
+  the output. Details in the tracker: https://tracker.ceph.com/issues/57460
+
+CephFS
+~~~~~~
+
+* CephFS: MDS evicts clients which are not advancing their request tids which
```

Comment:
```suggestion
* CephFS: it is now possible to pause write I/O and metadata mutations on a tree in the file system using a new suite of subvolume quiesce commands. This is implemented to support crash-consistent snapshots for distributed applications. Please see the relevant section in the documentation on CephFS subvolumes for more information.
* CephFS: MDS evicts clients which are not advancing their request tids which
```

I'm not sure how we missed this. Please add ^

Reply:
ACK